qualtran/simulation/xcheck_classical_quimb_test.py

```diff
@@ -42,7 +42,7 @@ def test_toffoli():
 
 
 def test_add():
-    add = Add(QUInt(bitsize=5))
+    add = Add(QUInt(bitsize=5), QUInt(bitsize=5))
```

Comment:
can you undo your change to this file and other test files to make sure the change is backward compatible?

Reply:
Yup I just finished going through and changing everything back.